service/src/main/resources/HTMLTemplate.html

```diff
@@ -467,304 +466,565 @@ <h3 class="heading heading--two">Attending the hearing</h3>
+                  <span class="u-block">{{child.value.additionalNeeds}}</span>
+                  {% if child.value.additionalNeedsDetails is not empty
+                  and child.value.additionalNeeds != 'No' %}
+                    <span class="u-block">{{ child.value.additionalNeedsDetails }}</span>
+                  {% endif %}
+                {% else %}
+                  -
+                {% endif %}
+              </dd>
+            </div>
+            <div class="data-list__row">
+              <dt class="data-list__title">Litigation capacity issues</dt>
+              <dd class="data-list__definition">
+                {% if child.value.litigationIssues is not empty %}
+                  {% if child.value.litigationIssues == 'YES' %}
+                    <span class="u-block">Yes</span>
+                    {% if child.value.litigationIssuesDetails is not empty %}
+                      <span class="u-block">{{ child.value.litigationIssuesDetails }}</span>
+                    {% endif %}
+                  {% elseif child.value.litigationIssues == 'NO' %}
+                    <span class="u-block">No</span>
+                  {% else %}
+                    <span class="u-block">Don't know</span>
+                  {% endif %}
+                {% else %}
+                  -
+                {% endif %}
+              </dd>
+            </div>
+            <div class="data-list__row">
+              <dt class="data-list__title">Confidential contact details</dt>
+              <dd class="data-list__definition">
+                {% if child.value.detailsHidden is not empty %}
+                  <span class="u-block">{{ child.value.detailsHidden }}</span>
+                  {% if child.value.detailsHiddenReason is not empty
+                  and child.value.detailsHidden != 'No' %}
+                    <span class="u-block">{{ child.value.detailsHiddenReason }}</span>
+                  {% endif %}
+                {% else %}
+                  -
+                {% endif %}
+              </dd>
+            </div>
+          </dl>
+        {% endif %}
+      {% set counter = counter + 1 %}
+      {% endfor %}
     {% endif %}
 
-    {% for child in case_data.children.additionalChildren %}
-      {% if child is not empty and child.value is not empty %}
+  {% elseif case_data.children1 is not empty and case_data.children is empty %}
+    {% set counter = 1 %}
+    {% for child in case_data.children1 %}
+      {% if child is not empty and child.value is not empty
+      and child.value.party is not empty %}
         <h3 class="heading heading--two">Child {{ counter }}</h3>
         <dl class="data-list">
           <div class="data-list__row">
             <dt class="data-list__title">Name</dt>
             <dd class="data-list__definition">
-              {% if child.value.childName is not empty %}
-                {{ child.value.childName }}
+              {% if child.value.party.firstName is not empty or
```

Comment:
why do you need this if?

Reply:
Because now that we've split the names. A user could enter just a last name, or just a first name.